test/test_retriever.py

```diff
@@ -165,10 +170,17 @@ def test_dpr_embedding(document_store, retriever):
 @pytest.mark.slow
 @pytest.mark.parametrize("retriever", ["retribert"], indirect=True)
 @pytest.mark.vector_dim(128)
-def test_retribert_embedding(document_store, retriever):
-
+def test_retribert_embedding(document_store, retriever, docs):
+    if isinstance(document_store, WeaviateDocumentStore):
+        document_store = WeaviateDocumentStore(
+            weaviate_url="http://localhost:8080",
+            index="haystack_test",
+            embedding_dim=128
+        )
+        document_store.weaviate_client.schema.delete_all()
+        document_store._create_schema_and_index_if_not_exist()
```

Comment:
Is this needed because of the dimensionality of the vector?

Reply:
Yes, exactly. Default is 768, which is set when WeaviateDocumentStore is initialized but we need 128 here.